chapter_02_repository.asciidoc

```diff
@@ -34,14 +34,15 @@ some code that looks more or less like <<api_endpoint_pseudocode>>:
 [role="skip"]
 [source,python]
 ----
-@flask.route.gubbbins
+@flask.route('/allocate')
```

Comment:
I shall privately mourn the loss of "gubbins" from our corpus.

Reply:
fear not! it is returned.